applications/nrf_desktop/doc/motion.rst

```diff
@@ -106,10 +116,15 @@ In these configurations, the module is a configuration channel listener and it p
 Implementation details
 **********************
 
+Implementation details depend on the selected implementation.
+
+Movement data from motion sensors
+=================================
+
```

Comment:
nit, can be ignored: I think that this selected and changing the headers related to motion sensor and heading style changes from line 127;136;147 could have been done in separate commit. Those changes relates to Motion Sensor module, not Motion Simulated one

Reply:
I modified the headers' ranks, because previously the section focused only on the motion sensor implementation and currently we want to have descriptions for other implementations next to it (introducing this change separately may not be best approach, because it's not needed unless description for the second implementation is introduced here).